src/index.js

```diff
@@ -41,7 +41,7 @@ try {
 }
 
 function getSbomFile(sbomFilePath) {
-  const _sbomFilePath = `.${sbomFilePath}`;
```

Comment:
this will always make the github action error out, if the user tries to specify an absolute path / or any path outside of current work dir. we shouldnt make any assumptions here.